cmd/interlock/run.go

```diff
@@ -163,18 +163,27 @@ func runAction(c *cli.Context) {
 		}
 	} else {
 		configPath := c.String("config")
+
 		if configPath != "" {
 			log.Debugf("loading config from: %s", configPath)
 
 			d, err := ioutil.ReadFile(configPath)
 			switch {
 			case os.IsNotExist(err):
+				log.Infof("Missing Interlock configuration: file=%s", configPath)
```

Comment:
You might also want to collapse this into a multiline string with a single `Error`.